docs/operating/enhanced.md

```diff
@@ -202,7 +198,12 @@ where
 * `POSTGRES_DB` - specifies the schema name that will generate inside your database
 * `POSTGRES_USER` - specifies your database username
 * `POSTGRES_PORT` - specifies your database's TCP port. Default is `5432`
-* `POSTGRES_STORAGE_CLASS` - specifies the database's storage class. Options are `"host"` if you set up your database in your local cluster; and `"efs"` if you already have an [AWS Elastic File System (EFS)](https://aws.amazon.com/efs/) storage that will keep your database records
+* `POSTGRES_STORAGE_CLASS` - specifies the database's storage class.
+    - `"host"` if you set up your database in your local cluster;
```

Comment:
```suggestion
    - `"host"` if you set up your database in your local cluster;
```
```suggestion
    - Set value to`"host"` if you set up your database in your local cluster;
```

Reply:
It it technical instruction, "Set value to" does not add semantic load